### PR TITLE
Tag v1.12.0-rc1

### PR DESCRIPTION
**Pull request by dominikschulz**
Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+## 1.12.0-rc1 / 2021-02-03
+
+* [BUGFIX] Add signature verification for updater
+* [BUGFIX] Allow using tilde
```
Comment (AnomalRoil) [inline CHANGELOG.md:4]: We might want to have clearer bugfixes in the future. I guess it's "in the config file", but anyway, not important

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+* [ENHANCEMENT] Add optional pinentry unescaping
+* [ENHANCEMENT] Add tpl funcs for Bcrypt and Argon2
+* [ENHANCEMENT] Add windows support to the self updater
+* [ENHANCEMENT] Confirm new age keyring passphrases
+* [ENHANCEMENT] KV secrets are now key-values, supporting multiple same key with different values
+* [ENHANCEMENT] UTF-8 emojis
+* [ENHANCEMENT] Use gpgconf to the the gpg binary
+* [ENHANCEMENT] Use main as the git default branch
+* [ENHANCEMENT] Use persistent SSH connections
+* [TESTING] Adding DI to Github Actions
```
Comment (AnomalRoil) [inline CHANGELOG.md:26]: Ups, my bad: should have been CD instead of DI 😅 
But not too important neither we can always edit the release page afterwards

**Review by AnomalRoil — APPROVED** [review]
Yay, let's do rc1 with our CD pipeline😃

File: CHANGELOG.md (hunk 1 above)
Comment (dominikschulz) [reply CHANGELOG.md:4]: Yes, these should be more specific ...

File: CHANGELOG.md (hunk 2 above)
Comment (dominikschulz) [reply CHANGELOG.md:26]: Yes, for a regular release I'd expect the person sending the PR to spend some time curating these entries (and the reviewer checking that they did).

But this is mostly for testing GHA, so ...

**Pull request merged**